servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/HttpObjectEncoder.java

```diff
@@ -258,8 +264,13 @@ private static void encodeChunkedContent(ChannelHandlerContext ctx, Buffer msg,
         if (contentLength > 0) {
             String lengthHex = toHexString(contentLength);
             ByteBuf buf = ctx.alloc().directBuffer(lengthHex.length() + 2);
-            buf.writeCharSequence(lengthHex, US_ASCII);
-            writeShortBE(buf, CRLF_SHORT);
+            try {
+                buf.writeCharSequence(lengthHex, US_ASCII);
+                writeShortBE(buf, CRLF_SHORT);
+            } catch (Exception e) {
```

Comment:
This code is unlikely to fail. Added try-catch for consistency, more safety, and better sleep :)